tests/ert_tests/ert3/console/integration/test_cli.py

```diff
@@ -282,7 +282,8 @@ def _assert_done_or_pending(captured, experiments, done_indices):
 @pytest.mark.requires_ert_storage
 def test_cli_status_no_runs(workspace, capsys):
     experiments_folder = workspace._path / _EXPERIMENTS_BASE
-    (experiments_folder / "E0").mkdir(parents=True)
+    (experiments_folder).mkdir(parents=True, exist_ok=True)
```

Comment:
I think we should avoid `exist_ok=True` here, since that might mean there is a problem where tests run in the same folder :thinking:

Reply:
I know that there was a test or a few failing when not specifying `exist_ok=True`, but might have been symptoms of other factors too. Trying without.